modules/mono/csharp_script.cpp

```diff
@@ -3536,6 +3536,18 @@ void CSharpScript::get_script_signal_list(List<MethodInfo> *r_signals) const {
 	}
 }
 
+bool CSharpScript::inherits_script(const Ref<Script> &p_script) const {
+	Ref<CSharpScript> cs = p_script;
+	if (cs.is_null()) {
+		return false;
+	}
+
+#ifndef _MSC_VER
+#warning TODO: Implement CSharpScript::inherits_script and other relevant changes after GH-38063.
+#endif
+	return false;
+}
```

Comment:
This should fix the Mono build and thus let CI pass.
Still needs an actual implementation, but since `get_base_script()` is also still a TODO, I'm not sure how to do it.